docs/advanced/peering/inter-cluster-network.md

```diff
@@ -26,9 +26,11 @@ The unpeer process will automatically remove the Liqo Gateway from the tenant na
 
 ## Manual on cluster couple
```

Comment:
Is it something like "option C"?
(previous subsections of type '##' started with "Option A" and "Option B".)

In any case, I don't understand what the title means.

Reply:
This title belongs to another page :)

Reply:
Ops... got it.
In any case, the title is complex to understand to me. I would simply say "Manual".

Reply:
Agree that the title is not so clear. Trying to make them more descriptive